### ISSUE TITLE
Snapshot capture function expects at most one argument

### ISSUE THREAD

**Issue by zipy124** [Parquery/pyicontract-lint#32]
The docs show how you can use snapshots to capture multiple arguments and combine them into one set. However when I try the same idea the code runs without error, but the linter throws the error:

```Snapshot capture function expects at most one argument, but got: ['account_id', 'account2_id'] (snapshot-invalid-arg)```

Does this linter error infer that the snapshot function should only ever take one argument? and if so why does the docs say otherwise?

Docs reference: https://icontract.readthedocs.io/en/latest/usage.html#snapshots-a-k-a-old-argument-values

Edit:
In-fact the linter fails on the documentation example 
```
Snapshot capture function expects at most one argument, but got: ['lst_a', 'lst_b'] (snapshot-invalid-arg)
```

**Comment by mristin** [comment]
Hi @zipy124,
Thanks for reporting this. I would say on first sight that it is a bug in the linter as I probably forgot to update it. I'll try to investigate the problem in the next few days.

Could you maybe send me a distilled short snippet of your code for which the linter fails so that I can include it in the test cases to avoid regressions?

**Comment by mristin** [comment]
Hi @zipy124 ,
I reproduced the bug on my side, it is a mistake in pyicontract-lint. This also revealed a couple of more errors, so thanks again a lot for this report, much appreciated!

**Comment by mristin** [comment]
@zipy124 I fixed the bug and released the patch version 2.1.2 to pypi: https://pypi.org/project/pyicontract-lint/. Please let me know if there are more issues.

**Comment by mristin** [comment]
(P.S. If you haven't done so, please star the project on GitHub to give it some visibility.)
